app/dao/__init__.py

```diff
@@ -1,6 +1,55 @@
 from sqlalchemy.exc import SQLAlchemyError
+from werkzeug.datastructures import MultiDict
+from sqlalchemy.orm.relationships import RelationshipProperty
+from app import db
 
 
 # Should I use SQLAlchemyError?
 class DAOException(SQLAlchemyError):
     pass
+
+
+class DAOClass(object):
```

Comment:
We could push some of this shared logic down onto the Model classes, and remove the DAO stuff maybe. For now lets not get into this we need the features more than the code refactors.